OneCommunityGlobal/HighestGoodNetworkApp
# Jonathan impl weekly summary color options

# Description
![PR #1 (1)](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/63978806/5b46d0c4-7b66-4a18-9069-51a1bc5367d8)
![PR #1 (2)](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/63978806/89b71c5d-2eed-46d6-b6d6-64d7f812f5e2)

## Main changes explained:
- Added 6 new summary options for the Weekly Summary Options dropdown inside of the Volunteering Tab and Create New User Form (E.g. Required, Not Required, Team Fabulous, Team Marigold, .etc)
- Changed the color of the summaries based on the user's Weekly Summary Option on the Weekly Summaries Reports page.


## How to test:
1. check into `jonathan_impl_weekly_summary_color_options`
2. do `npm install` and `npm run start:local` to run this PR locally

For Current User:
1. log as admin user
2. go to Profile page → Volunteer Times tab → Weekly Summary Options → Change Team
3. logged in as an Admin → Reports → Weekly Summary Reports
4. verify if summary text has changed color to the corresponding Weekly Summary Option

For New User:
1. log as admin user
2. go to Other Links → User Management → Create New User → Weekly Summary Options → Choose a Team
3. go to the _user you created_ Profile page → Volunteer Times tab
4. verify if Weekly Summary Option is the one you selected
5. log in as new user and submit a weekly summary
6. logged in as an Admin → Reports → Weekly Summary Reports
7. verify if summary text has changed color to the corresponding Weekly Summary Option


## Videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/63978806/561dc708-e42b-4684-a45c-e4cb3bd9701f


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/63978806/c5d25db7-45b0-4401-a714-e353473dd237
